### PR TITLE
Bug 1969538: Include VolumeSnapshotClass by default

### PR DESCRIPTION
**Pull request by huffmanca**
Allows the operator to create a default VolumeSnapshotClass.

### CODE REVIEW

**Comment by jsafrane** [comment]
/lgtm cancel

```
Operator degraded (AzureDiskCSIDriverOperatorCR_AzureDiskDriverStaticResourcesController_SyncError): AzureDiskCSIDriverOperatorCRDegraded: AzureDiskDriverStaticResourcesControllerDegraded: "volumesnapshotclass.yaml" (string): missing dynamicClient
AzureDiskCSIDriverOperatorCRDegraded: AzureDiskDriverStaticResourcesControllerDegraded: 
```

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -93,13 +99,19 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		kubeClient,
 		kubeInformersForNamespaces.InformersFor(defaultNamespace),
 		configInformers,
+		[]factory.Informer{
+			secretInformer.Informer(),
```
Comment (jsafrane) [inline pkg/operator/starter.go:103]: Is `secretInformer` useful? gcp-pd-csi-driver-operator uses it for hashing its secret, but Azure does not seem to use it.

Comment (huffmanca) [reply pkg/operator/starter.go]: I've gone and removed it here. I don't think we use it, as all of the secrets for Azure are in YAML files.

**Pull request merged**